lib/constants.ts

```diff
@@ -1,6 +1,6 @@
 export const SNOWDROP_ORG = "snowdrop";
 
-export const BOOSTER_GITHUB_TOPIC = "booster";
+export const BOOSTER_GITHUB_TOPIC = "example";
```

Comment:
I am pretty sure we also need to change: https://github.com/snowdrop/snowdrop-automation-client/blob/b4e3fa574cb3dea7a813e032670a37871cd29d51/lib/constants.ts#L14

Reply:
Which is why the tests are failing it seems

Reply:
Yes, I have modified it

Reply:
Yup, should be good!